internal/configs/named_values.go

```diff
@@ -38,9 +38,11 @@ type Variable struct {
 	ParsingMode VariableParsingMode
 	Validations []*CheckRule
 	Sensitive   bool
+	Deprecated  string
 
 	DescriptionSet bool
 	SensitiveSet   bool
+	DeprecatedSet  bool
```

Comment:
Does it make sense to set `deprecated` with an empty string? I am not sure if we need `DeprecatedSet` property.

Reply:
To be honest, didn't want to add `DeprecatedSet` either. Didn't really see the benefit of it, but I wanted to have it here just to spark the conversation about the standards here.
This seems more of a standards than an actual functional approach. I was looking at the `SensitiveSet` too and couldn't find any usage of that other than setting it again later to another attribute when merged.

I would like to remove the `DeprecatedSet` too and since you suggest the same, I will do that happily.

Reply:
It makes sense if we want to distinguish between the empty value and unset value. For example, we could differentiate whether sensitive was absent completely or set to false. Not sure if we use it in practice, but it still makes sense. For the deprecated field, I am not sure if we should allow `deprecated = ""`.